app/views/documents/_attachment.html.erb

```diff
@@ -13,8 +13,18 @@
   # to bust cache and view a specific edition, isn't applicable when publishing
   # to publishing-api.
   preview = controller.nil? ? nil : params[:preview]
+
+  attachment_data = {}
+  if attachment.html?
```

Comment:
We were going to make tracking be optional for anything inside the page content although this might be a special case. Did you look into making it optional and how complex was it?

Reply:
@andysellick I couldn't think of a way to toggle it - it might require a Ruby developer to implement, as the only way I could think of doing that was if there was a checkbox on the admin UI for HTML attachments which allowed you to turn off tracking. But then I guess there would also need to be some kind of comms between content and performance analysts to make sure tracking is only turned off for certain reasons?

Reply:
That's a good point. I wonder if this is the first time we've encountered this situation? Might be worth expanding our ruleset for how we implement this. Great explanation, let's keep it like this and have a wider discussion soon.